update/genesis/import.go

```diff
@@ -307,7 +341,11 @@ func (si *stateImport) importState(fileName string, trieKey string) error {
 	}
 
 	oldRootHash := value
-	log.Debug("old root hash", "value", oldRootHash)
+	log.Debug("old root hash", "value", oldRootHash, "read root hash", rootHash)
+
+	if bytes.Equal(value, trie.EmptyTrieHash) {
```

Comment:
You can use oldRootHash instead value, to be more clear here

Reply:
done